repos/cms_sw/cms_docker/githublabels.py

```diff
@@ -11,7 +11,7 @@
     TYPE_COMMANDS[arch] = [LABEL_COLORS["doc"], "%s-[0-9a-f]+" % arch, "mtype", True]
     TYPE_COMMANDS[arch + "-cms-docker"] = [
         LABEL_COLORS["doc"],
-        "%s-[a-z]+-(queued|building|done|error)" % arch,
+        "%s[-a-z]*-(queued|building|done|error)" % arch,
```

Comment:
@aandvalenzuela , this is still not correct as it can match `x86_64foobar-queued` i.e. tag part starting with `[a-z]` instead of `-`